Sources_v3/StreamChatUI/ChatChannel/MessageInputSlashCommandView.swift

```diff
@@ -87,7 +61,9 @@ open class MessageInputSlashCommandView: UIView {
         iconView.heightAnchor.constraint(equalToConstant: commandLabel.font.pointSize).isActive = true
     }
     
-    open func updateContent() {
-        commandLabel.text = "GIPHY"
+    override open func updateContent() {
+        commandLabel.text = commandName
+        
+        layer.cornerRadius = intrinsicContentSize.height / 2
```

Comment:
This should probably go to `layoutSubviews()`? The height can change and it's possible/likely that the actual height will be different than the proposed height by intrinsic content size.

Reply:
fixed in c33f8cd